executor/cook/config.py

```diff
@@ -32,22 +32,26 @@ def parse_time_ms(time_string):
             return 1000
 
     def __init__(self,
+                 checkpoint=0,
                  max_bytes_read_per_line=1024,
                  max_message_length=512,
                  memory_usage_interval_secs=15,
                  progress_output_env_variable=DEFAULT_PROGRESS_FILE_ENV_VARIABLE,
                  progress_output_name='stdout',
                  progress_regex_string='',
                  progress_sample_interval_ms=100,
+                 recovery_timeout='5mins',
```

Comment:
Nitpick: should we default to `15` minutes?

> recovery_timeout: Amount of time allotted for the agent to recover [Default: 15 mins].

http://mesos.apache.org/documentation/latest/agent-recovery/